origami.json

```diff
@@ -2,6 +2,7 @@
 	"description": "FT brand colours: Sass variables and helpers to use and define them",
 	"origamiType": "module",
 	"origamiCategory": "primitives",
+	"keywords": "colours, themes, schemes, pink, salmon, background, palette",
```

Comment:
How does this keyword search work, do we need to add the singular of colours and themes or will they still match?